src/hti/re_dash/database/impl/app_db.cljd

```diff
@@ -8,6 +8,7 @@
 
 (defonce app-db (atom {}))
 
+(defonce subscriptions-cache (atom {}))
```

Comment:
in re-frame there's a clear-subscription-cache function. from what i understand it's mostly there for dev-time hot reload.

wouldn't something like that be necessary for re-dash as well?